bolt_transport.go

```diff
@@ -255,6 +255,11 @@ func (t *BoltTransport) dispatchHistory(s *Subscriber, toSeq uint64) {
 			}
 		}
 		s.HistoryDispatched(responseLastEventID)
+		if !afterFromID {
+			if c := t.logger.Check(zap.ErrorLevel, "Can't find LastEventID in Bolt DB"); c != nil {
+				c.Write(zap.Error(fmt.Errorf("id not found %s", s.RequestLastEventID)))
```

Comment:
```suggestion
				c.Write(zap.String("LastEventID", s.RequestLastEventID)))
```

No need to log the Go error type here, just logging the ID in a structured field will be good enough.